### DISCUSSION TITLE
v19.19.5-3.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#148] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.19.5`  
Current YTLite Version: `3.0.1`  
Current App Compatibility: `iOS/iPadOS 14.0` or later

**RELEASE 19.19.5-3.0.1:** (2024-05-16)

- Updated Youtube from `19.18.2` to `19.19.5`
- Fixed #55 (Empty space when hiding Shorts) 
- Updated submodules
  
### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*
- VersionSpoofer (Lite) doesn't work with YTLite *(as the version is already spoofed by YTLite to fix YTClassicVideoQuality)*

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.19.5-3.0.1'>v19.19.5-3.0.1 - YTLitePlus</a>.</em>

**Comment by backstab5983** [comment]
Broken as reported here: https://github.com/YTLitePlus/YTLitePlus/issues/149

**Comment by S3bbz** [comment]
FYI: the .ipa link is shared in this release. Please remove it so Google doesn't issue a DMCA takedown.
Thank you for your great work :)

**Comment by arichornlover** [comment]
Hey @Balackburn I suggest adding in **YouTimeStamp** submodule/tweak!
YouTimeStamp is a tweak inspired off of YouTube ReVanced’s Version of the TimeStamp button.
so when the button is enabled in the Video Player, it will copy the url and includes the timestamp based on which part of the video you are watching.

that implementation would not have been possible without **bhackel**’s help!

**Reply by Balackburn** [reply]
The thing is YTLite already have this option when you click on pose it copy timestamp

**Reply by arichornlover** [reply]
YTLite already did it? 😭
I didn’t know.

**Reply by arichornlover** [reply]
> YTLite already did it? 😭 I didn’t know.

I checked the source-code, and it copies the timestamp by pressing the pause button?
YouTimeStamp uses the stuff from YTVideoOverlay to add custom buttons
but you don’t have to add it if YTLite’s already have one?

**Reply by Balackburn** [reply]
> > YTLite already did it? 😭 I didn’t know.
> 
> 
> 
> I checked the source-code, and it was copies the timestamp by pressing the pause button?
> 
> YouTimeStamp uses the stuff from YTVideoOverlay to add custom buttons
> 
> but you don’t have to add it if YTLite’s already have one?

Will add it

**Reply by Balackburn** [reply]
> but you don’t have to add it if YTLite’s already have one?

Hey arichorn sorry to ask you that but I don't have much free time to maintain YTLitePlus at this time, 
Would you mind doing some basic repo maintenance for the next few months? I really don't have the time right now.

**Reply by abheizthe** [reply]
> > but you don’t have to add it if YTLite’s already have one?
> 
> Hey arichorn sorry to ask you that but I don't have much free time to maintain YTLitePlus at this time, Would you mind doing some basic repo maintenance for the next few months? I really don't have the time right now.


sad to hear that if you're need to back to your activity's but, YtLite was updated with a lots of changes. 

Trying inject the deb from newest version of YtLite because my Yt app won't play any videos. 

with the latest build(/fork) from this repo (YtLitePlus 3.0.1 Yt 19.21.3) and some settings from YtLitePlus isn't working anymore. But, the app is always working

thank you for your hardwork to keep this things always working!
cheers!

----edit----

YtLitePlus settings still working, just need some adjustment

**Comment by samuelc02** [comment]
It's been about 2 months since I updated and the previous version I was using was 19.13.1-3.0.1, but since I updated to this new release I found a lot of settings gone, like disable snap to chapter and enable classic video quality menu. So I was just wondering is my build corrupt? Or were some functionalities removed for some reason in the past patches? But I didn't see any mention of them being removed after going through the past release notes.
